tests/test_integration.py

```diff
@@ -289,8 +289,8 @@ def test_send_many_traces(self):
         # register a single trace with a span and send them to the trace agent
         self.tracer.trace("client.testing").finish()
         trace = self.tracer.writer.pop()
-        # 30k is a right number to have both json and msgpack send 2 payload :)
-        traces = [trace] * 30000
+        # 20k is a right number to have both json and msgpack send 2 payload :)
```

Comment:
Just curious about this change.

Reply:
Adding the runtime id tag slightly increased the size of a span so now we need less span to have a split while uploading — that's why I had to decrease this number here to make the test still pass.